.github/actions/nm-build-vllm-whl/action.yml

```diff
@@ -24,11 +30,19 @@ runs:
       source $(pyenv root)/versions/${{ inputs.python }}/envs/${VENV}/bin/activate
       SUCCESS=0
       pip3 wheel --no-deps -w dist . || SUCCESS=$?
-      echo "status=${SUCCESS}" >> "$GITHUB_OUTPUT"
       ls -alh dist/
       BASE=$(./.github/scripts/convert-version ${{ inputs.python }})
       WHL_FILEPATH=$(find dist -iname "*nm_vllm*${BASE}*.whl")
       WHL=$(basename ${WHL_FILEPATH})
       echo "whl=${WHL}" >> "$GITHUB_OUTPUT"
+      # generate .tar.gz
+      $SUCCESS || python3 setup.py sdist || SUCCESS=$?
+      ls -alh dist/
```

Comment:
Not sure if this is intentional. Using the same `dist` folder includes the wheel in the tarball. May be use a different directory if it isn't intentional ? @mgoin

Reply:
The sdist argument is to generate source distribution, i.e. the tar.gz file. It'll be generated under the same folder as the wheel which is generated in an earlier step, i.e. dist/. This is the default output folder for generated packages. We want to keep them under dist/ since we'll upload them both to GH in one step by just providing the dist/ path to an upload action.